### PR TITLE
Grammar update in How This Course will Work

### PR DESCRIPTION
**Pull request by elimoises**
Screenshot 2022-01-09 032453

 - [x] You have read [The Odin Projects Contributing Guide](https://github.com/TheOdinProject/curriculum/blob/main/CONTRIBUTING.md).
 - [x] The title summarizes the change and where it happened, for example: "Fixes punctuation in Clean Code lesson".
 - [ ] If the PR is related to an open issue, use a [relevant keyword](https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests) and reference it with the `#` sign and the issue number.
 - [x] You have previewed your markdown (if any) in the [Odin Markdown Preview Tool](https://www.theodinproject.com/lessons/preview)
 - [ ] If changes are requested, please make the changes in a timely manner. After you submit the changes, request another review from the maintainer (top right).

#### 1. Describe the changes made and include why they are necessary or important:

...Fixed some grammatical errors in How This Course Will Work page
#### 2. Related Issue

Closes #XXXXX

### CODE REVIEW

File: foundations/introduction/how_this_course_will_work.md [hunk 1]
```diff
@@ -1,45 +1,44 @@
 ### Welcome to The Odin Project
 
-The Odin Project is an open source community dedicated to providing the best information sources to take you from zero to full-stack developer.  More information can be found at The Odin Project's [about page](https://www.theodinproject.com/about).
+The Odin Project is an open-source community dedicated to providing the best information sources to take you from zero to a full-stack developer.  More information can be found at The Odin Projects [about page](https://www.theodinproject.com/about).
```
Comment (twalton83) [inline foundations/introduction/how_this_course_will_work.md:3]: The original is correct here. This change makes "Project" plural.

File: foundations/introduction/how_this_course_will_work.md [hunk 2]
```diff
@@ -1,45 +1,44 @@
...
 
 ### Introduction
 
-In this unit, we'll learn about how the web works and start thinking about the basics of computer and web programming.
+In this unit, we’ll learn about how the web works and start thinking about the basics of computer and web programming.
```
Comment (twalton83) [inline foundations/introduction/how_this_course_will_work.md:7]: Is there a reason you're using a "smart quote" instead of a standard single quote?

File: foundations/introduction/how_this_course_will_work.md [hunk 3]
```diff
@@ -1,45 +1,44 @@
...
 
-If you are a non-English speaker, or English is not a primary language for you, this fact is not meant to discourage you, but to prepare you. 
+If you are a non-English speaker, or English is not a primary language for you, this fact is not meant to discourage you, but to prepare you.
 
-As another part of this preparation, we *highly* recommend spending extra time on topics and terms that you do not understand right away. We also encourage you to seek out additional resources that teach about these topics in your own native language so you can understand them more fully.
+As another part of this preparation, we highly recommend spending extra time on topics and terms that you do not understand right away. We also encourage you to seek out additional resources that teach about these topics in your native language so you can understand them more fully.
```
Comment (twalton83) [inline foundations/introduction/how_this_course_will_work.md:31]: Why did you remove emphasis here?

**Review by twalton83 — COMMENTED** [review]
I have some questions here.  I'm wondering if you used Word or some rich text processor instead of Markdown.

**Comment by elimoises** [comment]
I used Grammarly. I was practicing how to contribute. I'm sorry if I messed it up.

**Comment by twalton83** [comment]
You didn’t mess up anything, but this is part of the review process. Now, there were a few valid changes here, so I would pick through and address them, fix the smart quotes and add the emphasis back.

**Comment by elimoises** [comment]
I'm still new on this one and this was a good experience. thank you

**Comment by thatblindgeye** [comment]
@elimoises just wanted to reach out and check on the progress on the requested changes that have been made.

Also wanted to mention that the initial comment you made should be updated so that the PR title includes the lesson this update is made in ("Grammar update in How This Course Will Work" or something similar), and that all applicable check boxes are checked (just need to replace the space between the two brackets with an x, `[x]`).

**Comment by thatblindgeye** [comment]
@elimoises Can you give an update on the other requested changes that had been made? Otherwise this PR will be closed in 7 days.

**Comment by thatblindgeye** [comment]
Per previous comment this PR will be closed. If you would like this to be reopened to address previous comments/requests please feel free to comment on this PR again.
